Documentation/Storage-Configuration/Shared-Filesystem-CephFS/filesystem-storage.md

```diff
@@ -220,6 +220,168 @@ If the Rook cluster has more than one filesystem and the application pod is sche
+    resourceVersion: '49728'
+  persistentVolumeReclaimPolicy: Retain
+  storageClassName: rook-cephfs
+  volumeMode: Filesystem
+```
+
+* On this PV, change the `persistentVolumeReclaimPolicy` parameter to `Retain` to avoid it from being deleted when you will delete PVCs. Don't forget to change it back to `Delete` when you want to remove the shared volume (see full procedure in the next section).
+
+* Copy the YAML content of the PV, and create a new static PV with the same information and some modifications. From the original YAML, you must:
+  * Modify the original name. To keep track, the best solution is to append to the original name the namespace name where you want your new PV. In this example `newnamespace`.
+  * Modify the volumeHandle. Again append the targeted namespace.
+  * Add the `staticVolume: "true"` entry to the volumeAttributes.
+  * Add the rootPath entry to the volumeAttributes, with the same content as `subvolumePath`.
+  * In the `nodeStageSecretRef` section, change the name to point to the secret you created earlier, `rook-csi-cephfs-node-user`.
+  * Remove the unnecessary information before applying the YAML (claimRef, managedFields,...):
+
+Your YAML should look like this:
+
+```yaml
+kind: PersistentVolume
+apiVersion: v1
+metadata:
+  name: pvc-a02dd277-cb26-4c1e-9434-478ebc321e22-newnamespace
+spec:
+  capacity:
+    storage: 100Gi
+  csi:
+    driver: rook.cephfs.csi.ceph.com
+    volumeHandle: >-
+      0001-0011-rook-0000000000000001-8a528de0-e274-11ec-b069-0a580a800213-newnamespace
+    volumeAttributes:
+      clusterID: rook
+      fsName: rook-cephfilesystem
+      storage.kubernetes.io/csiProvisionerIdentity: 1654174264855-8081-rook.cephfs.csi.ceph.com
+      subvolumeName: csi-vol-8a528de0-e274-11ec-b069-0a580a800213
+      subvolumePath: >-
+        /volumes/csi/csi-vol-8a528de0-e274-11ec-b069-0a580a800213/da98fb83-fff3-485a-a0a9-57c227cb67ec
+      rootPath: >-
+        /volumes/csi/csi-vol-8a528de0-e274-11ec-b069-0a580a800213/da98fb83-fff3-485a-a0a9-57c227cb67ec
+      staticVolume: "true"
+    nodeStageSecretRef:
+      name: rook-csi-cephfs-node
+      namespace: rook
+    controllerExpandSecretRef:
+      name: rook-csi-cephfs-provisioner
+      namespace: rook
+  accessModes:
+    - ReadWriteMany
+  persistentVolumeReclaimPolicy: Retain
+  storageClassName: rook-cephfs
+  volumeMode: Filesystem
+```
+
+* In a new or other namespace, create a new PVC that will use this new PV you created. You simply have to point to it in the `volumeName` parameter. Make sure you enter the same size as the original PVC!:
+
+```yaml
+kind: PersistentVolumeClaim
+apiVersion: v1
+metadata:
+  name: second-pvc
+  namespace: newnamespace
+  finalizers:
+    - kubernetes.io/pvc-protection
+spec:
+  accessModes:
+    - ReadWriteMany
+  resources:
+    requests:
+      storage: 100Gi
+  volumeName: pvc-a02dd277-cb26-4c1e-9434-478ebc321e22-newnamespace
+  storageClassName: rook-cephfs
+  volumeMode: Filesystem
+```
```

Comment:
Please add a note here that cephcsi doesn't support snapshot, clone, resize, delete operations on the static pvc's. If required user needs to do it on the original PVC.

Reply:
Note added